src/main/java/org/lookout_studios/meals_management_system/meals_management_system/mysqlConnection.java

```diff
@@ -0,0 +1,30 @@
+package org.lookout_studios.meals_management_system.meals_management_system;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class MysqlConnection {
+    public void sqlConnection() {
```

Comment:
This method should be documented. Other than that, the name is not self-explanatory so I'm not exactly sure what is it that this method is doing